Hudl.Mjolnir/Command/DelegateAsyncCommand.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Hudl.Mjolnir.Command
+{
+    internal class DelegateAsyncCommand<TResult> : AsyncCommand<TResult>
+    {
+        private readonly Func<CancellationToken?, Task<TResult>> _func;
+
+        public DelegateAsyncCommand(string group, Func<CancellationToken?, Task<TResult>> func) : base(group, group, TimeSpan.FromMilliseconds(2000))
```

Comment:
Timeout here (and below) should reference the [default here](https://github.com/hudl/Mjolnir/blob/FuncCommands/Hudl.Mjolnir/Command/BaseCommand.cs#L30)